standalone-metastore/metastore-server/src/main/sql/derby/upgrade-3.2.0-to-4.0.0.derby.sql

```diff
@@ -18,6 +18,36 @@ UPDATE "APP"."TAB_COL_STATS" SET ENGINE = 'hive' WHERE ENGINE IS NULL;
 ALTER TABLE "APP"."PART_COL_STATS" ADD ENGINE VARCHAR(128);
 UPDATE "APP"."PART_COL_STATS" SET ENGINE = 'hive' WHERE ENGINE IS NULL;
 
+CREATE TABLE "APP"."SCHEDULED_QUERIES" (
```

Comment:
I am wondering whether you have tested all the upgrade scripts for the different RDBMS, e.g., with your docker image or the one that Alan created? I believe so but just to confirm with you, I know it is easy to have problems in some of the upgrade scripts that remain unnoticed for a while :)

Reply:
yes; I did when I was writing them - actually I've also spotted a bug which already had a ticket opened a week before....but the bug was actually there for around a year or so
I've opened HIVE-22302

Reply:
Cool, thanks (y)